spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -243,49 +243,6 @@
     end
   end
 
-  context 'LOA3 continuation' do
-    let(:user) { profile.user }
-    let(:otp) { 'abc123' }
-    let(:profile) do
-      create(
-        :profile,
-        deactivation_reason: :verification_pending,
-        phone_confirmed: phone_confirmed,
-        pii: { ssn: '6666', dob: '1920-01-01' }
-      )
-    end
-    let(:oidc_auth_url) do
-      client_id = 'urn:gov:gsa:openidconnect:sp:server'
-      state = SecureRandom.hex
-      nonce = SecureRandom.hex
-
-      openid_connect_authorize_path(
-        client_id: client_id,
-        response_type: 'code',
-        acr_values: Saml::Idp::Constants::LOA3_AUTHN_CONTEXT_CLASSREF,
-        scope: 'openid email profile:name social_security_number',
-        redirect_uri: 'http://localhost:7654/auth/result',
-        state: state,
-        prompt: 'select_account',
-        nonce: nonce
-      )
-    end
-
-    context 'phone verification' do
-      let(:phone_confirmed) { true }
-
-      it 'prompts to finish verifying profile, then redirects to SP' do
-        visit oidc_auth_url
-
-        sign_in_live_with_2fa(user)
-        enter_correct_otp_code_for_user(user)
-
-        redirect_uri = URI(current_url)
-        expect(redirect_uri.to_s).to start_with('http://localhost:7654/auth/result')
-      end
-    end
-  end
-
```

Comment:
This test and the one below in `spec/features/saml/loa3_sso_spec.rb` were removed because they are no longer valid for two reasons:

1. They are not passing in any phone number to `OtpDeliverySelectionForm`. The reason these tests passed before the changes in this PR is because when the form submission in `TwoFactorAuthenticationController#send_code` failed, it would redirect to `TwoFactorAuthenticationController#show`, which would send an OTP via SMS to the user. In real life, this would fail because the background job would attempt to send an SMS to a blank phone, but because the test was using the prefill code feature, it passed.

    With the validation changes in this PR, this test now fails as it should.

2. These tests are testing a feature that no longer exists. The feature that these tests are supposed to be testing is the one introduced in #1369. However, the way these tests were written, they're making a lot of assumptions and not really testing the feature in an end-to-end manner. We know this because we undid this feature in #1530, yet the tests continued to pass.